### ISSUE TITLE
Remove sm-versions of radio-button and checkbox from documentation

### ISSUE THREAD

**Issue by schaertim** [swisspost/design-system#3591]
The sm-versions are not supported in the v2 design and thus need to be removed from the docs as well.
```[tasklist]
### Tasks
- [ ] Remove small variant class styles for affected components
```

**Comment by gfellerph** [comment]
@gfellerph prepare discussion for all affected components

**Comment by gfellerph** [comment]
Resolved in #3710, small variants can be removed.

**Comment by gfellerph** [comment]
Closing as will be handled in component tickets.
